src/Bicep.Decompiler.IntegrationTests/Working/modules/main.bicep

```diff
@@ -23,7 +23,8 @@ module module1Deploy 'nested/module1.bicep' = {
   }
 }
 
-module module2Deploy 'nested/module2.bicep' = {
+module module2Deploy '?' /*TODO: replace with correct path to [variables('module2Url')]*/ = {

```

Comment:
Do you know why this happened? Should be possible to determine the correct path for the module here.

Reply:
Oops...the `.arm` and `.jsonc` are not copied to output folder. Fixed it by adding them as embedded resources to the project.